compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/SystemTheme.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui
+
+import androidx.compose.runtime.staticCompositionLocalOf
+import org.jetbrains.skiko.SystemTheme as SkikoSystemTheme
+import org.jetbrains.skiko.currentSystemTheme
+
+enum class SystemTheme {
+    Dark, Light, Unknown
+}
+
+val LocalSystemTheme = staticCompositionLocalOf {
+    currentSystemTheme.asComposeSystemTheme()
+}
+
+fun SkikoSystemTheme.asComposeSystemTheme() : SystemTheme {
```

Comment:
Might be private/internal

Reply:
I made it public because there are similar cases ([Paint](https://github.com/JetBrains/compose-multiplatform-core/blob/a103d7df9fdc02ff7e4ea91d8965f64893ce2d3b/compose/ui/ui-graphics/src/skikoMain/kotlin/androidx/compose/ui/graphics/SkiaBackedPaint.skiko.kt#L30), [Path](https://github.com/JetBrains/compose-multiplatform-core/blob/a103d7df9fdc02ff7e4ea91d8965f64893ce2d3b/compose/ui/ui-graphics/src/skikoMain/kotlin/androidx/compose/ui/graphics/SkiaBackedPath.skiko.kt#L32) and other classes from `androidx.compose.ui.graphics` package) where skiko-to-compose converters are public (maybe they used from other modules). This one can be internal for sure

Reply:
I also thing, that it is better change it to `private / internal`

Reply:
Made it private